online-survey-app/src/app/tangy-forms/tangy-forms-player/tangy-forms-player.component.ts

```diff
@@ -28,13 +28,18 @@ export class TangyFormsPlayerComponent implements OnInit {
   caseEventId: string;
   eventFormId: string;
   window: any;
+  actor: Record<string, any>;
```

Comment:
I think these property names need to be prefixed with xapi e.g. xapiActor, xapiEndpoint, etc to make it clear.